src/taipy/gui/builder/_element.py

```diff
@@ -106,24 +106,46 @@ def __init__(self, *args, **kwargs):
 class html(_Block):
     """A visual element defined as an HTML tag.
 
+    Use this class to integrate raw HTML to your page.
+
     This element can be used as a block element.
     """
 
     def __init__(self, *args, **kwargs):
         """Create a new `html` block.
 
-        TODO
+        Arguments:
+            args: A list of two unnamed arguments:
+                - *args[0]* is the HTML tag name. If empty or None, this represents an HTML text
+                  node.
+                - The text of this element.
+            kwargs: the HTML attributes for this element.
+
+        Examples:
+            - To generate `<br/>`, use:
+               ```
+               html("br")
+               ```
+            - To generate `<h1>My page title</h1>`, use:
+               ```
+               html("h1", "My page title")
+               ```
```

Comment:
No example of using kwargs?

Reply:
Well I had an example with 'id', but for some reason it's gone.
Thanks!

Reply:
Now I know why. Friday evening side effect.
Thanks again